r/vignettes/fs.Rmd

```diff
@@ -144,3 +147,28 @@ s3://minioadmin:minioadmin@?scheme=http&endpoint_override=localhost%3A9000
 
 Among other applications, this can be useful for testing out code locally before
 running on a remote S3 bucket.
+
+## Disabling the use of environment variables
+
+As mentioned above, it is possible to make use of environment variables to 
+configure access.  However, if you wish to pass in connection details via a URI
+or alternative methods but also have existing AWS environment variables defined,
+these may interfere with your session.  You can unset these environment 
+variables using `Sys.unsetenv()`, for example:
+
+```{r, eval = FALSE}
+Sys.unsetenv("AWS_DEFAULT_REGION")
+Sys.unsetenv("AWS_S3_ENDPOINT")
+```
+
+By default, the AWS SDK tries to retrieve metadata about user configuration, 
+which can cause conficts when passing in connection details via URI (for example
+when accessing a MINIO bucket).  To disable the use of AWS environment 
+variables, you can set environment variable `AWS_EC2_METADATA_DISABLED` to 
+`TRUE`.
+
+```{r, eval = FALSE}
+Sys.setenv(AWS_EC2_METADATA_DISABLED = TRUE)
+```
+
+
```

Comment:
```suggestion
```

```

Super minor

Reply:
Just checking - those backticks are required, right? Was it the trailing whitespace that needs removing?

Reply:
Oh oops, yes I was trying (and failing) to suggest that you drop one of the two new lines. Thanks for catching that!